config/locales/jobseekers.yml

```diff
@@ -42,6 +42,10 @@ en:
       applying_for_the_role_heading: Applying for the role
       applying_for_the_role_paragraph: Please complete the online application.
       current_step: Step %{current} of %{total}
+      banner_links:
+        apply: Apply for this job
+        draft: Started - continue application
```

Comment:
These hyphens look a bit stubby to me in the rendered page. Could you try them with en dashes and em dashes and see which you (or Chris) prefer?

https://en.wikipedia.org/wiki/Dash#Common_dashes_and_their_Unicode_code-points